src/components/admin/HearingToolbar.js

```diff
@@ -38,7 +36,7 @@ class HearingToolbar extends React.Component {
     ];
     const open = isOpen(hearing);
     if (open && hearing.published) {
-      statusLabel = <FormattedMessage id="published"/>;
+      statusLabel = (<Alert bsStyle="success"><Icon name="eye"/> <FormattedMessage id="published"/></Alert>);
```

Comment:
Oh, and did you style the (non-react) notify.js notifications too? The ones that pop up when you post things in the UI?